src/components/InactiveProjectListManager.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import {
+    Box,
+    Grid,
+    Switch,
+    FormGroup,
+    FormControlLabel
+} from '@material-ui/core'
+import { useQuery } from '@apollo/client';
+
+import LoadingProgress from './LoadingProgress'
+import { GET_INACTIVE_PROJECTS_COUNT } from '../operations/queries/ProjectQueries';
+
+const InactiveProjectListManager = ({
+    clientId,
+    showInactiveProjects,
+    setShowInactiveProjects
+}) => {
+
+    const { loading, error, data } = useQuery(GET_INACTIVE_PROJECTS_COUNT, {
+        variables: {
+            clientId: clientId ? Number(clientId) : null,
+        },
+        fetchPolicy: 'cache-and-network'
+    });
+
+    if (loading) return <LoadingProgress/>
+    if (error) return `Error! ${error.message}`;
+
+    return (
+        <Box
+            mt={3}
+            mb={3}
+            className='InactiveProjectListManager'
+        >
+            {data.getInactiveProjectsCount != 0
+                ? (
+                    <FormGroup>
+                        <FormControlLabel 
+                            control={
+                                <Switch 
+                                    checked={showInactiveProjects}
+                                    onChange={(event) => setShowInactiveProjects(event.target.checked)}
+                                    name='showInactiveProjects'
+                                    color='primary'
+                                />
+                            }
+                            label='Show inactive projects'
+                        />
+                    </FormGroup>
+                ) : null
```

Comment:
In this cases you can use the double ampersand operator https://es.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Idk how `null` renders on screen

Reply:
Thanks, Sofia I'm going to save that info because I always forget that I can do that